Source/WebCore/css/mediaControls.css

```diff
@@ -263,7 +263,7 @@ video::-webkit-media-text-track-region-container.scrolling {
 video::-webkit-media-text-track-display {
     position: absolute;
     overflow: hidden;
-    white-space: pre-wrap;
+    white-space: pre-line;
```

Comment:
Because this rule is more specific than the one in text-tracks.css, is it necessary to have?

Reply:
FYI - `mediaControls.css` is only used by Sony / Playstation port and I have highlighted it to @fujii that it would be good to explore `text-tracks.css` as single source so we can do this clean-up and remove this one. (Just mentioned and didn't discussed any future plans).